flamedisx/xenon/x1t_sr1.py

```diff
@@ -439,40 +443,62 @@ class SR1WallSource(fd.SpatialRateERSource, SR1ERSource):
       # TODO: The parameters here will need to be polished. 
       # They are fitted parameters and give a reasonable result.
       @staticmethod
-      def p_electron(nq, *, w_er_pel_a = -123. , w_er_pel_b = -47.7, 
-                    w_er_pel_c = 68., w_er_pel_e0 = 9.95):
+      def p_electron(nq, *, w_er_pel_c = 0.000174, lam = 0.015):
       
-          """Fraction of ER quanta that become electrons
-          Simplified form from Jelle's thesis
+          """Fraction of ER quanta that become electrons.
+          Exponentially falling yield.
           """
-          # The original model depended on energy, but in flamedisx
-          # it has to be a direct function of nq.
-          e_kev_sortof = nq * 13.7e-3
-          eps = fd.tf_log10(e_kev_sortof / w_er_pel_e0 + 1e-9)
-          qy = (
-              w_er_pel_a * eps ** 2
-              + w_er_pel_b * eps
-              + w_er_pel_c)
-          return fd.safe_p(qy * 13.7e-3)
+          # TODO: Maybe this can be tied to W in a combined fit
+          W = 13.8e-3 #keV
```

Comment:
The value to use should be the literature value, 13.7 eV. This is the posterior mean from 1T BBF fits.

Reply:
solved